### ISSUE TITLE
QNoise Licensing Question

### ISSUE THREAD

**Issue by LiamC12345** [DeiVadder/QNoise#1]
Is QNoise Free to use, modify and include in a commercial App?

I noticed you referenced sol-prog as someone who translated the original Java Perlin Noise Code to C++
But then it also says that you translated it to C++ as well.

The original Java Code is in the Public Domain but sol-prog has placed a GPU v3 License on his translation to c++ of Perlin Noise.

Is QNoise using entirely YOUR C++ translation from the original Java Code and thus not having any license placed on it OR
does it include work from sol-prog and thus bringing with it the GPU v3 License?

GPU v3 Licence means anything derived from OR that uses sol-prog's work must be an open source project, i.e not for use in a commercial app

In short - Can I use QNoise in a commercial App or is there a license attached to it?
